DEPENDENCIES.dash

```diff
@@ -0,0 +1,2 @@
+pypi/pypi/-/requests/2.31.0
```

Comment:
Why DEPENDENCIES file with .dash extension here?
Please make sure you have used dash command properly, you can use this [TRG](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-04/#checking-libraries-using-the-eclipse-dash-license-tool) Section for more information.